### PR TITLE
feat: Spawn electrs and esplora in fedimint-bin-tests

### PR DESCRIPTION
**Pull request by gotlougit**
This is done to partially address #2101 issue 5, and will help migrate the remaining portions of Fedimint's testing infra to Rust away from Bash scripts.

### CODE REVIEW

File: fedimint-bin-tests/src/main.rs [hunk 1]
```diff
@@ -544,13 +545,15 @@ async fn dev_fed(task_group: &TaskGroup, process_mgr: &ProcessManager) -> Result
     fed.await_gateways_registered().await?;
     info!("gateways registered");
     fed.use_gateway(&gw_cln).await?;
+    let electrs = Electrs::new(bitcoind.clone()).await?;
```
Comment (Maan2003) [inline fedimint-bin-tests/src/main.rs:549]: let do this concurrently with Lnd::new and Lightningd::new.

Comment (gotlougit) [reply fedimint-bin-tests/src/main.rs:549]: Sure!

**Review by Maan2003 — DISMISSED** [review]
LGTM

**Comment by Maan2003** [comment]
Feel free to do esplora as well. It should be very similar.

**Comment by gotlougit** [comment]
Now we spawn esplora in `fedimint-bin-tests` too, so this addresses subissue 4 in #2101

File: scripts/build.sh [hunk 2]
```diff
@@ -39,6 +39,7 @@ export FM_LND_DIR="$FM_TEST_DIR/lnd"
 export FM_BTC_DIR="$FM_TEST_DIR/bitcoin"
 export FM_CFG_DIR="$FM_TEST_DIR/cfg"
 export FM_ELECTRS_DIR="$FM_TEST_DIR/electrs"
+export FM_ESPLORA_DIR="$FM_TEST_DIR/esplora"
```
Comment (Maan2003) [inline scripts/build.sh:42]: also mkdir it bellow, just to be consistent.

**Review by Maan2003 — APPROVED** [review]
Thanks!

**Review by dpc — APPROVED** [review]

**Comment by justinmoon** [comment]
Great PR @gotlougit!

**Pull request merged**